source/quick_start_guide/index.html.md.erb

```diff
@@ -81,7 +83,7 @@ Make sure you [keep your API keys safe](/security/#securing-your-developer-keys)
 
 You can make test API calls to GOV.UK Pay with your test API key.
 
-You can use API testing tools such as <a href="https://www.getpostman.com/" target="_blank">Postman</a> or <a href="https://insomnia.rest/" target="_blank">Insomnia REST</a> to make test API calls.
+You can use API testing tools such as <a href="https://www.getpostman.com/">Postman</a> or <a href="https://insomnia.rest/">Insomnia REST</a> to make test API calls.
```

Comment:
Any reason we don't want to open these links in a new window ?

Reply:
It's best avoided, especially as it can cause accessibility problems: see the Design System guidance here - https://design-system.service.gov.uk/styles/typography/#opening-links-in-a-new-tab